vagrant/roles/local.prep/tasks/main.yml

```diff
@@ -3,7 +3,7 @@
     src: .vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory
     dest: ./vagrant_ansible_inventory
 
-- name: addapt inventory - ssh options
+- name: adapt inventory - ssh options
```

Comment:
This hunk could / should have been a separate commit.
The fact that you mentioned this in your commit message shows that you thought about it. Do it next time! :-)